src/handlers/handlers.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+var path = require('path');
+
+var handlers = (module.exports = {});
+
+handlers.serveLanding = function(request, response) {
+  fs.readFile(
+    path.join(__dirname, '..', '..', 'public', 'index.html'),
+    function(err, file) {
+      if (err) {
+        throw err;
```

Comment:
Throws have been changed to the same format as the rest (yes, it was an oversight). 

RE: Solutions branches, if you as maintainers want a different format for solutions I'm happy to acquiesce to that, I was just under the impression that solutions/mentor notes branches were the "done" thing. Hadn't seen foundersandcoders/master-reference#456.

Lock files are part of a node repo. npm explicitly tells you that they are supposed to be committed, my opinion would be that unless we as an organisation explicitly decide they are _not_ to be committed then we should conform to what npm suggest. As for "other stuff" idk what you mean, sorry.